packages/orbit-components/src/Timeline/TimelineStep/components/TimelineStepDesktop.tsx

```diff
@@ -31,7 +31,10 @@ const TimelineStepDesktop = ({
 
   return (
     <Stack inline shrink direction="column" align="center">
-      <div className={cx("relative flex w-[calc(100%+theme(spacing.400))] items-center")}>
+      <div
+        className={cx("relative flex w-[calc(100%+theme(spacing.400))] items-center")}
+        aria-hidden
```

Comment:
this is applied only for desktop, mobile/tablet views still read the "image".

<img width="494" alt="image" src="https://github.com/user-attachments/assets/9ababa33-ad6d-4c03-ae56-7308303c4610" />

Reply:
Right. What does `obrázek` mean btw? 😃

Reply:
that means "image" :D I have VoiceOver in Czech :D

Reply:
Thanks, TIL. I ended up adding the aria-hidden to the icon. I did not remove it from the div in Desktop because it would still announce the timeline as a group and repeat the label, which is weird 🤷‍♂️